opensoundscape/torch/models/utils.py

```diff
@@ -240,13 +241,22 @@ def tensor_binary_predictions(scores, mode, threshold=None):
         if threshold is None:
             raise ValueError(f"threshold must be specified for multi_target prediction")
         # predict 0 or 1 based on a fixed threshold
```

Comment:
since it's not necessarily obvious what happens in the code here, I would update this comment (something like #if single value is supplied for threshold use it for all classes. if list of thresholds is provided, each value in the list will be used a threshold for corresponding class.